perl6.html.markdown

```diff
@@ -4,6 +4,7 @@ language: perl6
 filename: learnperl6.pl
 contributors:
     - ["vendethiel", "http://github.com/vendethiel"]
+    - ["samcv", "https://gitlab.com/u/samcv/"]
```

Comment:
That's totally fine, I just saw you were supposed to add yourself as contributor.